index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Joel's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    
+    <div class="wrapper">
+    <header class="main-body">
+        <div class="tools">
+            <nav class="main-nav">
+                <div class="items">
+                    <a href="#">My Logo</a>
+                    <div id="harmburger">
+                        <img alt="harmburger" src="images/harmburger_img.png">
+                    </div>
+                </div>
+                <!-- <div class="col-item">
+                    <div><a href="#portfolio">Portfolio</a></div>
+                    <div><a href="#about">About</a></div>
+                    <div><a href="#contact">Contact</a></div>
+
+                </div> -->
+                <div class="menu"><a href="#" class="mainMenu"><i class=""></i></a></div>
+
+
+            </nav>
+
+        </div>
+
+    </header>
+    <main class="main-section">
+        <section class="head-part">
+            <h1>I'm Joel <br>Glad to see<br> you!</h1>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don't hesitate to
+            contact me.</p>
+
+                <span class="paragraph">LETS CONNECT</span>
+                <div class="social_icons">
+                    <ul class="font_icons">
+                        <li>
+                            <a href="#">
+                                <img alt="twitter" src="images/twitter-icon.png"/>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img alt="link" src="images/Linkedin-icon.png"/>
+                            </a>
```

Comment:
Great design @Joel-100 🤝

- [ ] Kindly align the elements of the main section with `display: flex` as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md).